### ISSUE TITLE
Проводить чемпионат в два круга

### ISSUE THREAD

**Issue by alexeypodolian** [football-it-league-dnipro/football-it-dp-ua#41]
У команд нет права на ошибку, когда чемпионат проводится в один круг.

**Comment by twichez** [comment]
и при этом не увеличивать размер взноса в два раза

**Comment by Vizhan** [comment]
и при этом не увеличивать размер взноса ~~в два раза~~

**Comment by steamhammer** [comment]
Сейчас в каждой лиге по 10 команд. Два круга это 18 игр. Вместе с кубковыми выйдет 20+. 
20+ игр это 18-20 недель. Почти полгода. По-моему многовато.

**Comment by alexeypodolian** [comment]
Есть и другие варианты: 
1) Сделать меньше команд в лиге и увеличить количество лиг
2) Убрать кубок (в этом сезоне кубка не было)

**Comment by steamhammer** [comment]
Даже без кубка 10 команд в одной лиге сыграют между собой все игры как минимум за 5 месяцев (а с учетом переносов, форс-мажоров и т.п. выйдет и того больше).

**Comment by twichez** [comment]
и в чем проблема играть столько времени?

**Comment by steamhammer** [comment]
в том, что:
а) это долго и утомительно
б) пропорционально возрастают затраты, как материальные, так и моральные
в) каких-то ощутимых преимуществ длинный сезон не приносит. Ну кроме меньшей цены ошибки. :)

Опытным путем мы выяснили, что оптимальная длина сезона - 9-12 игр на каждую команду.

**Comment by alexeypodolian** [comment]
Три лиги по 7 команд?

**Comment by steamhammer** [comment]
Это интересная идея. Но так перераспределить команды мы сможем только не раньше следующего сезона (осень 2020). 
В качестве альтернативы можно подумать о смешанной системе: две группы по 5 команд и играть между собой в два круга, а потом по кубковой системе, "на вылет".

**Comment by twichez** [comment]
1. 11 лиг города играют и все ещё неутомимы
2. Возростают затраты на что?
3. Приносит больше эмоций, опыта и игровой практики 

Крайне сомневаюсь в результатах ваших опытов, так как тысячи людей играют по подобной схеме и всех все устраивает

**Comment by steamhammer** [comment]
Вместе посчитаем количество игр в каждой из 11 лиг города? или просто сойдемся на том что они не играют каждый чемпионат по полгода?

Каждая игра имеет свою цену. Она складывается из стоимости аренды поля, услуг судей, услуг оператора и т.п. Чем больше игр - тем больше стоимость.

На количество эмоций длина сезона не влияет. Даже наоборот - меньше игр, больше накала.

**Comment by twichez** [comment]
По 8 команд в лиге, играют по 14 игр, и нормально. Играют на тех же полях, с прямой трансляцией и комментатором, при это взнос меньше чем в ИТ лиге, другое дело что организаторы требуют большие суммы, а взамен не могут качественно организовать турнир.

О эмоциях каждый судит субъективно, для одних радость только в поднятии кубка над головой, вторые получают удовольствие от процесса и от каждой игры в частности.

Можно сколько угодно здесь все обсуждать, но все бесполезно, потому что решать все равно будут организаторы, а то на что они способны я уже увидел по предыдущим турнирам.

**Comment by steamhammer** [comment]
Прямая трансляция, комментатор, фотограф у них входят в стоимость взноса? 

На что "способны организаторы" все и так знают. Если условия нашего турнира устраивают - милости просим. Если нет, ну что ж, каждому свое. Никто никого принуждать не будет.

**Comment by twichez** [comment]
да, можно обратиться к организаторам городского турнира для уточнения как у них получается при лучших условиях брать меньше денег.

"На что "способны организаторы" все и так знают. Если условия нашего турнира устраивают - милости просим. Если нет, ну что ж, каждому свое. Никто никого принуждать не будет."
Это проявление слабости:) Здесь все создано для высказывания пожеланий, предпочтений, негодований. А сейчас еще раз подтверждаются мои слова о том, что организаторы не способны принимать критику и менять что то в лучшую сторону.

**Comment by steamhammer** [comment]
Пожелания и предложения приветствуются. Неконструктивная критика и откровенное искажение фактов - нет.
Мне вот очень интересно узнать про "лучшие условия за меньшие деньги" в городской лиге. Т.к. насколько мне известно в стоимость входит только фотограф, но не видеотрансляции с комментатором. Может быть я что-то пропустил?

**Comment by EdgarGaytsuk** [comment]
в суперлиге, 1 и 2 лиге входит все, но и цена в 2 раза дороже остальных

**Comment by twichez** [comment]
В прошлом сезоне за 9 игр на хюмане в манеже взнос с коментатором и онлайн трансляцией в высшей лиге был около 10 тысяч. А какой был взнос в айти?) О чем тут ещё говорить))

Не говоря уже о том, что обзоры на некоторые матчи скинули через месяц после окончания чемпа.

**Comment by steamhammer** [comment]
А сколько стоит взнос в городской лиге сейчас?

**Comment by steamhammer** [comment]
Впрочем, эта issue заводилась для обсуждения другого вопроса (чемпионат в два круга).
По всем остальным вопросам, плз, заводи отдельную.

**Comment by twichez** [comment]
Не важно какой взнос сейчас, потому что они играют в зале.

**Comment by steamhammer** [comment]
Окей окей. Как скажешь :)

**Comment by anton-kochev** [comment]
1. Проводить Чемпионат и Первенство лиги можно в два круга при условии:
- количество команд в лиге будет 6, максимум 7, что дает нам 10-12 матчей за сезон (это 3-3.5 месяцев);
- количество команда в лиге 8 и более, но мы уходим от проведения двух сезонов в год и делаем один длинный сезон;

2. К вопросу о "ненасытности" участников футболом. На предложение провести Зимний Кубок (длинною 5-6 матчей) отозвались две команды. Конкретно эта ситуация говорит нам о том, что спрос на дополнительный футбол крайне невелик.

3. Важно услышать больше двух мнений по вопросу формата соревнований для того, чтобы принять взвешенное решение.

**Comment by OleksiiKipra** [comment]
летом у большей части игроков начинаются отпуска и собрать команду на игру становится очень сложно. растягивать сезон на лето так себе идея.
учитывая количество команд, практика двух сезонов в год в один круг отлично подходит IT лиге.
двойной круг будет оправдан при малом количестве команд в лиге. не думаю, что много команд согласятся уйти в третью лигу или насильно покинуть высшую, просто чтобы уменьшить там кол-во команд.

**Comment by steamhammer** [comment]
@OleksiiKipra переформатироваться в три лиги по 6-7 команд можно только по результатам сезона. Т.е. перед стартом четко объявить - этот сезон играем в две лиги, потом делаем три лиги.

Насчет лета я полностью согласен. В прошлых весенних сезонах в июне уже начинались пропуски и переносы из-за отпусков.

Играть длинный сезон с осени на весну не очень комфортно из-за холодов. Играть в зале ИМХО не так приятно как на искусственной поляне.

Одним словом, слишком много минусов, при этом плюсов от длинного сезона почти нет (по крайней мере я не вижу). Если будет большой запрос со стороны участников - можно рассматривать, иначе считаю что затея не особо нужная.

**Comment by twichez** [comment]
на какое число запланирован старт чемпионата?

**Comment by steamhammer** [comment]
На вторую половину марта. Либо 14-е, либо 21-е.
